apps/snitch_core/test/data/model/wishlist_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule Snitch.Data.Model.WishListTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+
+  import Snitch.Factory
+
+  alias Snitch.Data.Model.WishListItem
+
+  setup do
+    user = insert(:user)
+    [user: user]
+  end
+
+  setup :variants
+
+  @tag variant_count: 1
+  test "add item to wishlist", context do
```

Comment:
Name should indicate that failure is expected.

Reply:
This has clauses for both failure and success, so no such indication in name.